packages/uix-host-react/package.json

```diff
@@ -19,7 +19,7 @@
   "bugs": "https://github.com/adobe/uix-sdk/issues",
   "dependencies": {
     "@adobe/uix-core": "^0.8.4",
-    "@adobe/uix-host": "^0.8.4"
+    "@adobe/uix-host": "file:.yalc/@adobe/uix-host"
```

Comment:
Need to remove `yalc` dependencies (and check package.lock)

Also, need to up versions of changes packages.

Reply:
Why do we use yalc within SDK? My assumption was we use yalc to link dev packages from different repositories

Reply:
package `uix-host-react` uses `uix-host` as a dependency (which was, in my case, not updated automatically despite being in same monorepo) so I had to add this `uix-host` yalc dependency to `uix-host-react` package.